src/test/java/com/google/cloud/tools/appengine/api/deploy/StageArchiveConfigurationTest.java

```diff
@@ -34,12 +34,12 @@ public void setUp() {
     // files are files and directories are directories?
     // should we use paths instead?
     configuration =
-        StageFlexibleConfiguration.builder(file, file, file).dockerDirectory(file).build();
+        StageArchiveConfiguration.builder(file, file, file).dockerDirectory(file).build();
```

Comment:
The arguments here are a little weird, the same thing four times? but I guess that's not new